### PR TITLE
Initial styling

### PR DESCRIPTION
**Pull request by jmevans0211**
## What does this PR do?
This PR gets some basic styling done. Adding colors, fonts, and alignment.

### Where should the reviewer start?
- Home screen

### How should this be manually tested?
- Pull down the branch and check it out. Click through the nav bar (with all the categories near the top) and just note of anything where the design seems off.
-If you don't feel like pulling down that image in this PR might be fine.

### Card(s) closed:
**N/A**

### Screenshots:
_(item highlighted orange is the cursor location)_
Screen Shot 2020-02-27 at 11 30 35 AM

### Additional notes/questions:
- Have a few things I'm not quite sold on that I'd love your help with:
a. **How to better that masonry layout.** It looks fine at the homepage, but if there are less than 5 or 6 recipes they hug the left side. Not the biggest deal. Users will be uploading their own images that will be all the same width, but not the same height. This can be changed though. 🧐
b. `EvansEats` **font** at the top
c. Color of **plus sign** in upper right
d. I like the idea of the **recipe cards**, but think there's a way to make it a bit nicer looking. Would love to hear your thoughts on this one especially. 👩🏼‍🔬

Thanks, @aripp2 !

### Helpful Resources
- https://sass-lang.com/documentation/at-rules/import (see `Heads Up` section)

### CODE REVIEW

File: src/containers/Header/Header.js [hunk 1]
```diff
@@ -22,30 +22,33 @@ export class Header extends Component {
...
 
   render () {
+    const categories = [
+      {title: 'Apps', class: 'apps'},
+      {title: 'Soups and Salads', class: 'soups_and_salads'},
+      {title: 'Meals', class: 'meals'},
+      {title: 'Beverages', class: 'beverages'},
+      {title: 'Desserts', class: 'desserts'}
+    ];
+
```
Comment (aripp2) [inline src/containers/Header/Header.js]: Great job refactoring and cleaning up this code!

File: src/containers/Header/Header.scss [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  h1 {
+    margin: .5em .5em .05em .5em;
+  }
+  h3 {
+    margin: .75em 1.5em .5em .75em;
+    transform: translateY(30%)
+  }
+}
+
+.header__nav--ul {
```
Comment (aripp2) [inline src/containers/Header/Header.scss]: I believe if you use `<NavLink>` in the header component you will have an active class that you can style here or the styling can happen right in the component. I would have to check the docs to be sure, but just something to think about and look in to.

**Comment by aripp2** [comment]
Over all I think it is looking great! I added a few comments where I think you can incorporate some active states to improve the UX.

Here are my thoughts on your questions:

a. How to better that masonry layout:

- I think you are on the right track here. I have always struggled with the masonry layouts. I did notice how items were "hugging" the left side. I feel like this troubleshooting comes down to a lot of trial an error. I also agree that the more recipes that are added the better it will look.

b. EvansEats font at the top:

- I LOVE the font here and the color scheme of the entire application. I would make this a clickable element, using navigation/routing to return "home" where the user can view all of the recipes. I noticed that was the default view when I when to the page but couldn't find a way to get back to that view. (This was advice I received on a previous project).

c. Color of plus sign in upper right:

- I think this color works well, as you continue working building out the functionality for it be sure to incorporate a hover state, and consider having some indication to the user what will happen when they click on it. You can add hover text that says something along the lines of `Click to add a new recipe`

d. I like the idea of the recipe cards, but think there's a way to make it a bit nicer looking. Would love to hear your thoughts on this one especially. 👩🏼‍🔬

- So I agree with you on this. It doesn't look bad but I see areas for improvement. I noticed with the lighter ones, I believe the Macaroons is an example of this, they almost blend into the background/card header too much. I like using a slight box shadow that complements the color scheme to help with this, and/or maybe just a tad darker for the border.

**Comment by jmevans0211** [comment]
Thanks so much for your feedback @aripp2 !!!

**Pull request merged**